src/main/java/seedu/address/model/meeting/Meeting.java

```diff
@@ -24,17 +26,20 @@ public class Meeting {
 
     private Description description;
     private Set<Note> notes = new HashSet<>();
+    private ArrayList<Contact> participants = new ArrayList<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Meeting(Title title, Time time, Place place, Description description, Set<Note> notes) {
-        requireAllNonNull(title, time, place, description, notes);
+    public Meeting(Title title, Time time, Place place, Description description,
+                   Set<Note> notes, ArrayList<Contact> contacts) {
+        requireAllNonNull(title, time, place, description, notes, contacts);
         this.title = title;
         this.time = time;
         this.place = place;
         this.description = description;
         this.notes.addAll(notes);
+        this.participants.addAll(contacts);
```

Comment:
For readability, perhaps we can standardise the name of the attribute/argument to be either `participants` or `contacts` only?